devicetypes/smartthings/dawon-zwave-smart-switch.src/dawon-zwave-wall-smart-switch.groovy

```diff
@@ -0,0 +1,392 @@
+		commands << zwave.configurationV1.configurationSet(parameterNumber: 2, size: 1, scaledConfigurationValue: humidityOffset as int)
+	}
+	commands << zwave.configurationV1.configurationGet(parameterNumber: 1)
+	commands << zwave.configurationV1.configurationGet(parameterNumber: 2)
+
+	commands << zwave.sensorMultilevelV5.sensorMultilevelGet(sensorType: 0x01) // temperature
+	commands << zwave.sensorMultilevelV5.sensorMultilevelGet(sensorType: 0x05) // humidity
+
+	sendCommands(commands,1000)
+}
+
+/**
+ * Mapping of command classes and associated versions used for this DTH
+ */
+private getCommandClassVersions() {
+	[
+		0x20: 1,  // Basic
+		0x25: 1,  // Switch Binary
+		0x30: 1,  // Sensor Binary
+		0x31: 5,  // Sensor MultiLevel
+		0x32: 3,  // Meter
+		0x56: 1,  // Crc16Encap
+		0x60: 3,  // Multi-Channel
+		0x70: 2,  // Configuration
+		0x84: 1,  // WakeUp
+		0x98: 1,  // Security
+		0x9C: 1,  // Sensor Alarm
+		0x71: 3   // Notification
+	]
+}
+
+def parse(String description) {
+	log.info("parse called: description: ${description}")
+	def result = []
+	def cmd = zwave.parse(description)
+	if (cmd) {
+		result = zwaveEvent(cmd)
+	}
+	return createEvent(result)
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.basicv1.BasicReport cmd, endpoint=null) {
+	log.info "BasicReport called: "+cmd +endpoint
+	def value = cmd.value ? "on" : "off"
+	endpoint ? changeSwitch(endpoint, value) : []
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.switchbinaryv1.SwitchBinaryReport cmd, endpoint = null) {
+	log.info "SwitchBinaryReport called: ${endpoint}, ${cmd.value}"
+	def value = cmd.value ? "on" : "off"
+	endpoint ? changeSwitch(endpoint, value) : []
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.securityv1.SecurityMessageEncapsulation cmd, endpoint = null) {
+	log.info "SecurityMessageEncapsulation called: ${cmd}, ${endpoint}"
+	def encapsulatedCommand = cmd.encapsulatedCommand(commandClassVersions)
+	if (encapsulatedCommand) {
+		waveEvent(encapsulatedCommand)
+	} else {
+		log.warn "Unable to extract encapsulatedCommand from $cmd"
+		createEvent(descriptionText: cmd.toString())
+	}
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.multichannelv3.MultiChannelCmdEncap cmd, endpoint = null) {
+	log.info "MultiChannelCmdEncap called: '${cmd}'  endpoint '${endpoint}'"
+	def encapsulatedCommand = cmd.encapsulatedCommand()
+	log.debug "MultiChannelCmdEncap: encapsulatedCommand '${encapsulatedCommand}'"
+	zwaveEvent(encapsulatedCommand, cmd.sourceEndPoint as Integer)
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.notificationv3.NotificationReport cmd, endpoint = null) {
+	log.info "NotificationReport called: notificationType: '${cmd.notificationType}', event: '${cmd.event}', endpoint '${endpoint}'"
+	def result = []
+
+	if (cmd.notificationType == 0x08) {
+		def value = cmd.event== 0x03? "on" : "off"
+		endpoint ? result = changeSwitch(endpoint, value) : []
+	} else if (cmd.notificationType == 0x04) {
+		def map = [name: "tamper"]
```

Comment:
you have not declared the `"Tamper"` capability.

Reply:
I also think this part not needed in now. So I will remove that "else if (cmd.notificationType == 0x04)...".